libdnf5/conf/config_main.cpp

```diff
@@ -113,6 +113,7 @@ class ConfigMain::Impl {
     Config & owner;
 
     OptionNumber<std::int32_t> debuglevel{2, 0, 10};
+    OptionNumber<std::int32_t> errorlevel{3, 0, 10};
```

Comment:
This could be removed if get_error_level() accessors were were implemented as a dummy `return 0`.

Reply:
I don't know how to easily do that.
The getters return a reference and I don't think those have a null value.

I have kept it for now.